app/views/playlists/_related_items.html.erb

```diff
@@ -22,9 +22,14 @@ Unless required by applicable law or agreed to in writing, software distributed
         <tr>
           <% position = clip.playlist_position(@playlist.id) %>
           <td class='clip_title'>
-            <%= link_to playlist_path(@playlist, position: position) do %>
-              <%= clip.title %>
-            <% end %>
+            <a
```

Comment:
Will this link be able to be copy+pasted by a user?

Reply:
@adamjarling is working on that for a different story.

Reply:
Ok, I didn't realize that was in scope of that story, but that makes sense.  I'll add a note to it to make sure it is explicit.

Reply:
I've got a fix in place to make these links copyable. The URL in the address bar will still need to be updated when transitioning to different playlist item ala #2536.